Lib/glyphsLib/builder/axes.py

```diff
@@ -403,9 +415,8 @@ def set_user_loc(self, master_or_instance, value):
                         pass
             return
 
-        # For masters, set directly the custom parameter (old way)
-        # and also the Axis Location (new way).
-        # Only masters can have an 'Axis Location' parameter.
```

Comment:
you modified this comment removing "Only masters can have an 'Axis Location'" but the code around the comment is the same as before; when master_or_instance is an instance and not a master, the code above returns early and skips the following block. So I'm not sure whether the comment or the code should be right

Reply:
The code doesn't differentiate between master and instance, so as I understand it, it has set the custom parameter on masters and instances alike all along.

Reply:
No, look at the `if hasattr(master_or_instance, "instanceInterpolations"):` block just above; that is True for any GSInstance, by definition (if I an not mistaken) and it `return`s early thus skipping anything that comes afterwards which is then assumed to only apply for non-instances, hence masters...

Reply:
checking for `instanceInterpolations` is quite weak. Wouldn’t be `isinstance(master_or_instance, GSInstance)` be better?

Reply:
yes probably.. i'm not sure to be honest why @belluzj did it like that.